lib/screens/login/login_view_model.dart

```diff
@@ -16,4 +18,36 @@ class LoginViewModel extends StateNotifier<void> {
   bool isValidPassword(String? password) {
     return !(password == null || password.length < 8);
   }
+
+  login({required String email, required String password}) async {
```

Comment:
Missing trailing commas, please avoid setting the type of function is `dynamic (or empty)`, we can use `void` or `Future<void>` 🙏